_includes/tutorials/aggregating-count/confluent/code/tutorial-steps/dev/input-events.json

```diff
@@ -0,0 +1,9 @@
+"Die Hard":{"title":"Die Hard","sale_ts":"2019-07-18T10:00:00Z","ticket_total_value":12}
```

Comment:
Do the numbers need to be in quotes now?  e.g., `12` --> `"12"`

Reply:
@ybyzek great question, you are a thorough reviewer! They don't need to be in quotes here. The input is an integer and the output will be a string. The output used to be a `Long` but `ccloud` CLI does not support `Long` as a value format (I verified in #cli slack).

Reply:
👍 Input=Long, Output=String.  Thanks for the explanation